modules/exploits/multi/http/horde_turba_file_upload.rb

```diff
@@ -0,0 +1,182 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name'            => 'Horde Turba File Upload Vulnerability',
+      'Description'     => %q{
+          Horde Groupware Webmail contains a flaw that allows an authenticated remote
+          attacker to execute arbitrary PHP code. The exploitation requires the Turba
+          subcomponent to be installed. This module was tested on versions 5.2.22 and 5.2.17.
+        },
+      'License'         => MSF_LICENSE,
+      'Author'          =>
+        [
+          'Ratiosec',
+        ],
+      'References'      =>
+        [
+                    ['CVE', '2019-9858'],
+                    ['URL', 'https://www.ratiosec.com/2019/horde-groupware-webmail-authenticated-arbitrary-file-injection-to-rce/'],
+        ],
+      'DisclosureDate'  => 'Mar 24 2019',
+      'Platform'        => 'php',
+      'Arch'            => ARCH_PHP,
+      'Targets'         => [
+    ['Automatic', { }],
+    ['PEAR', { 'path': '/var/www/html/'}],
+    ['Ubuntu', { 'path': '/usr/share/horde/' }],
+    ],
+      'DefaultTarget'   => 0
+    ))
+
+    register_options(
+      [
+        OptString.new('TARGETURI',  [true, 'The base path to the web application', '/']),
```

Comment:
Pretty sure `TARGET_URI` is the convention.

Reply:
Nope:

```
root@kali-2018:~/Desktop/metasploit-framework# grep -rn TARGETURI modules/ | grep OptString | wc -l
465
root@kali-2018:~/Desktop/metasploit-framework# grep -rn TARGET_URI modules/ | grep OptString | wc -l
3
```